spec/spec_helper.rb

```diff
@@ -23,3 +23,6 @@ def coverage_needed?
   config.order = :random
   Kernel.srand config.seed
 end
+
+# Require supporting ruby files from spec/support/ and subdirectories.  Note: engine, not Rails.root context.
```

Comment:
I believe the note at the end of the comment can be deleted since it doesn't apply in this context.